tools/sgen/input/FluentUIStyle.yml

```diff
@@ -247,17 +255,21 @@ AP_MSFListTokens:
   disclosureInterspace: $Spacing.xxSmall
   disclosureSize: $Icon.size.small
   horizontalCellPadding: $Spacing.medium
-  iconColor: $Colors.Foreground.neutral4
   iconInterspace: $Spacing.medium
-  iconSize: [ default: $Icon.size.medium, large: $Icon.size.xxlarge ]
   labelColor: $Colors.Foreground.neutral1
   labelFont: $Typography.body
+  leadingViewColor: $Colors.Foreground.neutral4
+  leadingViewSize: $Icon.size.medium
   sublabelColor: $Colors.Foreground.neutral3
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
+  trailingItemSize: $Icon.size.medium
+
+MSFListCellSmallTokens extends MSFListCellTokens:
```

Comment:
I guess what I meant on defining the different sizes for the leading view size was more on the lines of the backgroundColor in line 250, for example:

` trailingItemSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxlarge]`

Reply:
Oops! I see. Let me check with Victor on these, trailing items should only be 1 size AFAIK

Reply:
Custom trailing items are 1 size excluding specific components (button/toggle), so I'll leave this alone for now.